### ISSUE TITLE
Atom-typescript server is not started

### ISSUE THREAD

**Issue by dyping** [TypeStrong/atom-typescript#1500]
Atom: 1.37.0-beta1 X64
OS: Mac OS X 10.14.5 Beta
GUI: atom-ide-ui

1. When I open a pure JS project, there is no popup type in the mouse hover variable in any JS file. Use the shortcut key `command + shift + P` to execute `Typescript Initialize Config` before it can be used normally.

2. When I open a JS, TS hybrid project, if only one JS file is opened, the phenomenon is the same as 1; if a TS file is opened, it works normally.

**Comment by lierdakil** [comment]
In English please.

**Comment by dyping** [comment]
1. When I open a pure JS project, there is no popup type in the mouse hover variable in any JS file. Use the shortcut key `command + shift + P` to execute `Typescript Initialize Config` before it can be used normally.

2. When I open a JS, TS hybrid project, if only one JS file is opened, the phenomenon is the same as 1; if a TS file is opened, it works normally.

**Comment by lierdakil** [comment]
Thanks.

Please see https://github.com/TypeStrong/atom-typescript/blob/master/docs/faq.md#i-want-to-use-atom-typescript-with-javascript-too

TL;DR: Atom-TypeScript is not auto-started on JavaScript because some users might not want that. Some additional configuration is required.

FWIW, I'd prefer to avoid the "additional configuration" step myself, but it doesn't seem possible due to Atom limitations.

**Comment by dyping** [comment]
Thanks.
